src/plugins/Plugin.js

```diff
@@ -89,6 +89,18 @@ module.exports = class Plugin {
     }
   }
 
+  setMetaFromTargetForm () {
+    const el = findDOMElement(this.opts.target)
+    const formMetaData = getMetaFromForm(el)
+    if (formMetaData) {
+      this.core.log('Adding metadata from form:')
+      this.core.log(formMetaData)
+      this.core.setMeta(formMetaData)
+    } else {
+      this.core.log('Couldn’t extract metadata from form')
+    }
+  }
```

Comment:
What about putting it in `mount()` method? We have that for all UI plugins anyway, and that’s where we run `findDOMElement(target)`, makes sense to attempt extracting meta before mounting? Tried that in the recent commit, let me know what you think :)